marklogic-junit5/build.gradle

```diff
@@ -14,26 +14,23 @@ repositories {
 }
 
 dependencies {
-  compile project(":marklogic-unit-test-client")
-	compile "com.marklogic:ml-javaclient-util:3.11.0"
-	compile "jaxen:jaxen:1.1.6"
-	compile "org.junit.jupiter:junit-jupiter-api:5.4.1"
-	compile "org.junit.jupiter:junit-jupiter-params:5.4.1"
-	compile "org.springframework:spring-context:5.0.8.RELEASE"
-	compile "org.springframework:spring-test:5.0.8.RELEASE"
-
-	// Support for DHF is provided, but a client must specify their own version of the DHF library to use
-	compileOnly ("com.marklogic:marklogic-data-hub:4.2.2") {
-		exclude module: "ml-javaclient-util"
-	}
+  implementation project(":marklogic-unit-test-client")
```

Comment:
This needs to be "api" - a user of marklogic-junit5 may extend AbstractMarkLogicTest, in which case they have a compile dependency on these classes.